src/test/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacNumericValidatorTest.java

```diff
@@ -605,6 +606,34 @@ public final void isaacNumericValidator_WrongChoiceType_ExceptionShouldBeThrown(
         validator.validateQuestionResponse(numericQuestion, new Choice());
     }
 
+    /*
+     Test displayUnit overrides requiresUnits.
+    */
+    @Test
+    public final void isaacNumericValidator_TestInconsistentDisplayUnitsOverride_UnitsIgnored() {
+        // Set up the question object:
+        IsaacNumericQuestion someNumericQuestion = new IsaacNumericQuestion();
+        someNumericQuestion.setRequireUnits(true);
+        someNumericQuestion.setDisplayUnit("SOME-FAKE-UNIT");
+
+        List<Choice> answerList = Lists.newArrayList();
+        Quantity someCorrectAnswer = new Quantity(correctIntegerAnswer, "SOME-FAKE-UNIT");
```

Comment:
Doesn't the fact that you've set the display unit mean that you don't need to specify the correct unit in the Quantity answer here? Just checking to make sure I've understood the change.

Reply:
Yes. This tests that even though the unit is incorrect, the response is still correct because it is ignored. But actually, now you make me consider it again, we should give an error if you provide a unit when you shouldn't have done? And not just silently ignore it? An error is probably more consistent with everything else!

Reply:
I think this test should stay as is, it's testing that a unit-less student answer is marked correct despite the fact a question specifies displeyUnit and requireUnits. You did throw a content error if requireUnits and displayUnit are both set stating requireUnits would be overwritten, so I think that's error enough?

Reply:
Ah, yes, I think I muddled what I wanted to do and what I actually did! I agree with Ben that this is fine as-is!